### PR TITLE
fix duplicate task keys

### PR DESCRIPTION
**Pull request by jakemac53**
closes https://github.com/dart-lang/mono_repo/issues/8

### CODE REVIEW

**Comment by jakemac53** [comment]
tests coming...

File: mono_repo/lib/src/travis_command.dart [hunk 1]
```diff
@@ -24,6 +24,7 @@ class TravisCommand extends Command {
 }
 
 Future generateTravisConfig({String rootDirectory}) async {
+  rootDirectory ??= p.current;
```
Comment (kevmoo) [inline mono_repo/lib/src/travis_command.dart]: I think I only pass the directory into getTravisConfig, right? Which handles null...

Not sure looking here...

**Comment by jakemac53** [comment]
Ya there is a test! which validates the broken behavior, lol :).

Anyways it still fails with this change so working on fixing it

**Comment by kevmoo** [comment]
Users are great...

File: mono_repo/lib/src/travis_command.dart (hunk 1 above)
Comment (jakemac53) [reply mono_repo/lib/src/travis_command.dart]: We also use this here, so the test was passing (we pass in a rootDirectory) but it actually was failing  when you went through the real command.

Comment (kevmoo) [reply mono_repo/lib/src/travis_command.dart]: Riiight...

**Pull request merged**